src/stories/Feedback.stories.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import FeedbackItem from '@/components/feedback/FeedbackItem';
+
+// fixture
+import {
+  feedbackFirst,
+  feedbackSecond,
+  feedbackThird,
+  badFeedback,
+} from './fixture/feedback';
+
+export default {
+  title: 'Feedback Item',
+  component: FeedbackItem,
+} as ComponentMeta<typeof FeedbackItem>;
+
+const Template: ComponentStory<typeof FeedbackItem> = (args) => (
+  <FeedbackItem {...args} />
+);
```

Comment:
요거 React.createElement 함수 사용하면 props spreading 없이 동일한 동작 구현할 수 있습니다.

- https://github.com/telbby/telbby-frontend/pull/52#discussion_r767253065

```suggestion
const Template: ComponentStory<typeof FeedbackItem> = (args) => React.createElement(FeedbackItem, args);
```

Reply:
저는 요거 린트 끄는 것 어떤가 하고 리뷰 달았습니닷!!

Reply:
린트 꺼서 다른 PR에 올려두었습니다 !